### ISSUE TITLE
Customizable catch phrase to run a workflow

### ISSUE THREAD

**Issue by ibnesayeed** [nwtgck/actions-comment-run#152]
It will be cool if we can have multiple workflow files equipped with comment runner, but have different setups (e.g., one with all the dependencies and services installed to run an app and other with minimal code when certain commands do not requite a full-fledged setup). Then configure each workflow to be invoked with different catch phrases.

One way to implement it would be to change hard-coded `@github-actions run` catch phrase into an input variable with the existing value as the default. This way, the system will continue to function as it does, but advanced users can configure it to trigger different workflows with custom catch phrases.

**Comment by nwtgck** [comment]
If I understand your request correctly,

You can use `if: contains(github.event.comment.body, '<my-super-environment-1>')` and you can comment like the following.

````md
@github-actions run
<my-super-environment-1>'

```sh
#! /bin/sh
# Do something
```
````

**Comment by ibnesayeed** [comment]
Yes, it is certainly a possible workaround, but with added boilerplate noise in the comment. What you are suggesting is basically a global master catch phrase with added second-level filter phrase, which will be the responsibility of the user to ensure that the routine is not triggered because actual Action code only checks for a specific hard-coded phrase. I personally like the "batteries included, but replaceable" philosophy.
